rdkcentral/rdkservices
# RDK-43786: Implement Thunder Plugin Configuration for CMake-3.20 & above (#4490)

* RDK-43786: Implement Thunder Plugin Configuration for CMake-3.20 & above (#4411)

Reason for change: updated the rdkservices plugins config based on modern config generator Test Procedure: Verify in Jenkin Build
Risks: High
Signed-off-by: Thamim  Razith <tabbas651@cable.comcast.com>
(cherry picked from commit cbbd3c2ffecd5d74cd8bdc37f9c64ded75535dd9)

SERXIONE-3290: Unable to play content/launch major apps

Reason for change: OCDM socket path was done only for Platco/Hisense platform , we need to remove the connector socket Test Procedure: Verify in Jenkin Build
Risks: High
Signed-off-by: Thamim  Razith <tabbas651@cable.comcast.com>
(cherry picked from commit 27f7152c4fea9cec6b0a42b42480a4e88c24fab4)

* RDK-43786: Implement Thunder Plugin Configuration for CMake-3.20 & above (#4432)

Reason for change: updated the Network.conf.in and PersistentStore.conf.in Test Procedure: Verify in Jenkin Build
Risks: High
Signed-off-by: Thamim  Razith <tabbas651@cable.comcast.com>
(cherry picked from commit bbc49ce3aa0c0fc8168e818077afb7399ecdf7ed)

RDK-43786: Implement Thunder Plugin Configuration for CMake-3.20 & above (#4431)

Reason for change: added the webkitbrowser-plugins conf.in for modern config generator
Test Procedure: Verify in Jenkin Build
Risks: High
Signed-off-by: Thamim  Razith <tabbas651@cable.comcast.com>

Co-authored-by: Karunakaran A <48997923+karuna2git@users.noreply.github.com>
(cherry picked from commit 339b397c649b8a49b59940a51fe05427bcd3b0f4)

* RDK-43786: Implement Thunder Plugin Configuration for CMake-3.20 & above (#4455)

Reason for change: Updated the startuporder configuration in conf.in into each rdkservices plugin's Test Procedure: Verify in Jenkin Build
Risks: High
Signed-off-by: Thamim  Razith <tabbas651@cable.comcast.com>
(cherry picked from commit 879658e49ded63d624bafe595fd1536e844c8573)

RDK-43786: Implement Thunder Plugin Configuration for CMake-3.20 & ab… (#4456)

…ove (rdkcentral#4455)

Reason for change: added the JSPP.conf.in based on modern generator and also to avoid Sercomm-Xione-US-Flex2.0 Device Build compilation Error Test Procedure: Verify in Jenkin Build
Risks: High
Signed-off-by: Thamim  Razith <tabbas651@cable.comcast.com>
(cherry picked from commit adadca1c85add27d94b17461f022fdb847c3a99c)

* RDK-43786: Implement Thunder Plugin Configuration for CMake-3.20 & above

Reason for change: Updated the Webkitbrowser conf.in based on those config file Test Procedure: Verify in Jenkin Build
Risks: High
Signed-off-by: Thamim  Razith <tabbas651@cable.comcast.com>
---------

Co-authored-by: tabbas651 <74683978+tabbas651@users.noreply.github.com>
Co-authored-by: Thamim Razith <ThamimRazith_AbbasAli@comcast.com>
(cherry picked from commit 4c4caa2a75deeb87454bcd263f9fbd0b68973fd4)
